app/index.jade

```diff
@@ -176,9 +179,27 @@ head
   
   <!-- @if PRODUCTION !>
   link(rel='stylesheet', href='css/application.css')
-  script(src='js/application.min.js')  
   <!-- @endif -->
+
+body(ng-controller="AppCtrl", ui-view="body")
   
+  //- This does not depend on Angular and will show as soon as the images are done loading.
+  <!-- @if PRODUCTION !>
+  .blocket-loading
+    div
+      img.full(src="/img/blocket-full-4847e61888d2d3c9f33f14840f92783d1c91dea2.png")
+      img(src="/img/blocket-no-boosters-c3f2a027ddfdbbce8eba8fc5da84c65d89d14fbc.png")
+  <!-- @endif -->  
+
+  noscript
+    img(src="/img/logo-bw.png")
+    h3 Please enable JavaScript to use the Blockchain Wallet
+
+  //- Load application.min.js in the body async.
+  <!-- @if PRODUCTION !>
+  script(src='js/application.min.js')  
```

Comment:
If you want to load javascript in the body on production, you should do the same in dev. I'll add a commit.